test/functional/wallet_reorgsrestore.py

```diff
@@ -88,7 +88,7 @@ def run_test(self):
         # Node0 wallet file is loaded on longest sync'ed node1
         self.stop_node(1)
         self.nodes[0].backupwallet(self.nodes[0].datadir_path / 'wallet.bak')
-        shutil.copyfile(self.nodes[0].datadir_path / 'wallet.bak', self.nodes[1].chain_path / self.default_wallet_name / self.wallet_data_filename)
```

Comment:
In commit "Fix wallet directory initialization" (17b4955e4b1e4f02f00a948b432f18caa9b250a5)

Is there a reason this test is changing? I'd expect the change to be backwards compatible so existing tests should not need to change

Reply:
This is also because of the change in the wallet dir init logic. The `wallet/ `dir is created when a new node is started and no wallet is detected in data_dir